### ISSUE TITLE
beginner_source/nn_tutorial.py 오탈자 및 내용 수정

### ISSUE THREAD

**Issue by Angela-Park-JE** [PyTorchKorea/tutorials-kr#851]
## 문서 URL
_수정이 필요한 URL을 남겨주세요. (예. https://tutorials.pytorch.kr/beginner/basics/intro.html)_
- **URL**: https://tutorials.pytorch.kr/beginner/nn_tutorial.html#torch-nn-functional

## 변경 사항
_(1)어떤 단어 / 문장 / 내용이 (2)어떻게 변경되어야 한다고 생각하세요?_

* 코드 문자로 처리되어 있거나 괄호로 원어를 병치하였거나 링크로 연결되는 특수한 문자열 뒤에 (붙여 써야 할) 조사가 이어지는 경우는 따로 다루지 않았습니다. 강조의 의미나 링크, 병치에 대한 가시성을 고려한 의도일 수도 있다고 생각했습니다. 다만 일관성이 부족하여 이 부분을 논의 후 통일하면 좋을 것 같습니다.
* 또한 전달력 높고 명확하며 깔끔한 질 좋은 번역에 감사하며 오탈자 찾기를 진행하였습니다. 다만 맞춤법 설명을 표기법 그대로 참조한 부분들이 많아서 다소 경직된 어조의 오탈자 리뷰가 되었음을, 어수룩한 후배 보듯 넓은 마음으로 봐주시길 양해를 구합니다.

`# torch.nn 이 실제로 무엇인가요?`

- 클릭해주세요. : 클릭해 주세요.
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.
	- [한국어 어문 규범 - 제47항 보조 용언 띄어쓰기](https://www.korean.go.kr/kornorms/regltn/regltnView.do?regltn_code=0001&regltn_no=267#a267)
	- 제47항에 따르면 보조 용언은 띄어 씀을 원칙으로 하되, 경우에 따라 붙여 씀도 허용한다고 합니다. 
	- 그러나 링크의 1) 을 참고하자면 앞말에 조사가 붙거나 앞말이 합성 용언인 경우, 그리고 중간에 조사가 들어갈 적에는 그 뒤에 오는 보조 용언은 띄어 쓰는 것이 원칙이므로, '클릭+하다'처럼 합성 용언은 뒤의 보조 용언을 띄어 쓰는 것이 맞는 것으로 보입니다. 물론, 본용언이 합성어나 파생어라도 그 활용형이 2음절인 경우에는 붙여 쓴 말이 너무 긴 것은 아니므로 본용언과 보조 용언을 붙여 쓸 수 있다고 합니다. (ex. 구해 본다/구해본다)

- PyTorch 는 : PyTorch는
	- 원어에 쓸 때에도 조사를 붙여 쓰므로 수정하면 좋을 것 같습니다.
	- [한국어 어문 규범 - 제41항 조사 띄어쓰기](https://www.korean.go.kr/kornorms/regltn/regltnView.do?regltn_code=0001&regltn_no=261#a261)

- 피쳐도 : 특징도
	- [TRANSLATION_GUIDE.md](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md)에 따르면 [원문](https://pytorch.org/tutorials/beginner/nn_tutorial.html)의 feature는 "특징"으로 번역하기로 약속하였으므로 수정하면 좋을 것 같습니다.
	- "피쳐를"도 "특징을"로 수정하면 좋을 것 같습니다.

- 그리고나서 : 그러고 나서
	- '그러고'는 '그리하고'가 줄어든 말로 보조동사인 '나다'가 동사 뒤에서 '-고 나다'의 구성으로 쓰인 형태입니다. 보조동사도 보조 용언으로 제47항에 따라 띄어쓰는 것이 원칙인 것으로 보입니다.

- 한번에 : 한 번에
	- 단위를 나타내는 명사는 띄어 씁니다.
	- [한국어 어문 규범 - 제43항 단위 명사 띄어쓰기](https://www.korean.go.kr/kornorms/regltn/regltnView.do?regltn_code=0001&regltn_no=263#a263)

- 피쳐를 : 특징을
	- 마찬가지로 원문에서 feature라고 등장하는 것을 확인하였으며, 번역 가이드에 따라 수정하면 좋을 것 같습니다.

`## MNIST 데이터 준비`

- 것 입니다. : 것입니다.
	- 서술격 조사 '이다'는 앞말과 붙여 씁니다. '이다'는 '이고', '이니', '인', '이죠', '입니다' 등의 형태로 

- pickle 을 : pickle을
	- 조사를 붙여 쓰므로 수정하면 좋을 것 같습니다.

- 형태 이고, : 형태이고,
	- 서술격 조사 '이다'는 앞말과 붙여 씁니다. '이다'는 '이고', '이니', '인', '이죠', '입니다' 등의 형태로 활용합니다.
	- [표준국어대사전 - "이다"3](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%EC%9D%B4%EB%8B%A4)

- 하나를 : 한 장을  
	- "Let’s take a look at one" 원어 표현을 살리셨습니다. 다만 이미지 하나를 살펴보자는 의미이므로 조금 더 자연스럽도록 "한 개를" 혹은 "한 장을"이라고 하는 것도 좋아 보입니다.

- 살펴 봅시다; : 살펴봅시다;
	- 표준국어대사전에 등재된 합성어로 붙여 씁니다.
	- [표준국어대사전 - "살펴보다"](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%EC%82%B4%ED%8E%B4%EB%B3%B4%EB%8B%A4)

- 배열 보다는 : 배열보다는
	- 앞말이 비교의 기준이 되는 대상임을 나타내는 부사격 조사 '보다'는 앞말과 붙여 써야 합니다.
	- [표준국어대사전 - "보다"2](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%EB%B3%B4%EB%8B%A4)

- _우리는_ 데이터를 변환해야 합니다. : 데이터를 변환해야 합니다.
	- "we", "우리"라는 번역을 지워도 좋을 것 같습니다.

`## (torch.nn 없이) 밑바닥부터 신경망 만들기`

- 계산해주는 : 계산해 주는
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.

- 기억해주세요. : 기억해 주세요.
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.

- 점검해봅시다, : 점검해 봅시다.
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.

- 우리의 무작위 모델에 대한 손실을 _점검해봅시다, 그럼으로써_ _우리는_ 나중에 역전파 이후에 개선이 있는지 확인할 수 있습니다. : 우리의 무작위 모델에 대한 손실을 _점검해 봄으로써_ 역전파 이후 개선이 있는지 나중에 확인할 수 있습니다. 
	- "Let’s check our loss with our random model, so we can see if we improve after a backprop pass later."
	- 영어식의 쉼표와 we라는 주어에 대한 부분을 고려하여 조금 다듬어 보았습니다. 

- 목표값 : 목푯값
	- [한국어 어문 규범 - 제30항 사이시옷](https://www.korean.go.kr/kornorms/regltn/regltnView.do?regltn_code=0002&regltn_no=423#a423) 사이시옷 규정에 따라, '목푯값'이 옳습니다.
		1. 순우리말로 된 합성어 또는 순우리말과 한자어로 된 합성어로서 앞말이 모음으로 끝난 경우
			1) **뒷말의 첫소리가 된소리로 나는 것** : 값이 "-깝"으로 소리납니다.
			2) 뒷말의 첫소리 ‘ㄴ,ㅁ’ 앞에서 ‘ㄴ’ 소리가 덧나는 것
			3) 뒷말의 첫소리 모음 앞에서 ‘ㄴㄴ’ 소리가 덧나는 것
		2. 두 음절로 된 다음 한자어
			곳간, 셋방, 숫자, 찻간, 툇간, 횟수

- 매 반복마다, _우리는_ 다음을 수행할 것입니다 : 매 반복마다, 다음을 수행할 것입니다.
	- we라는 원어가 큰 의미 없으므로 번역하지 않아도 될 것 같습니다. we라는 원어 표현 번역을 지양하기로 한 것으로 알고 있습니다.

- 매니져 : 매니저
	- 국립국어원 외래어 표기 규정에 따라 '매니져'의 옳은 표기는 '매니저'입니다.
	- [표준국어대사전 - "매니저"](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%EB%A7%A4%EB%8B%88%EC%A0%80)

- 여기 에서 : 여기에서
	- 조사는 그 앞말에 붙여 씁니다.

- 그러고나서 : 그러고 나서
	- 띄어쓰기하는 것이 바른 표현입니다. (앞에 같은 예시가 있었습니다.) 

- 되버립니다. : 되어버립니다./돼버립니다.
	- 'ㅚ'로 끝나는 용언 어간에 어미 '-어(-었)'가 준 형태로 '돼버립니다.'를 씁니다. 《한글 맞춤법》 제35항에서 용언의 어간 'ㅚ' 뒤에 '-어'가 어울려 '-ㅙ'로 될 적에 준 대로 적도록 규정하고 있습니다.
	- [한국어 어문 규범 - 제35항 준말 붙임2](https://www.korean.go.kr/kornorms/regltn/regltnView.do?regltn_code=0001&regltn_no=255#a255)

- 사용해보세요. : 사용해 보세요.
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.

-	확인해봅시다. : 확인해 봅시다.
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.

- _우리는_ 제일 간단한 : 제일 간단한
	- "we’ve created and trained a minimal neural network..."
	- we라는 원어가 큰 의미 없으므로 번역하지 않아도 될 것 같습니다. we라는 원어 표현 번역을 지양하기로 한 것으로 알고 있습니다.  
	- 그러나 뒤이어 나오는 "We expect that the loss will have decreased and accuracy to have increased, and they have."에서의 "우리는"은 문서 초반에 환경 세팅에 대해 설명할 때처럼, 기대하고 있는 주체인 독자와 저자를 "우리"라고 명시해도 충분히 자연스럽기 문에 두는 것이 좋다고 생각합니다.

`# torch.nn.functional 사용하기`

- 이제 _우리는_ 코드를 리팩토링(refactoring) 하겠습니다, 그럼으로써 : 이제 코드를 리팩토링(refactoring) 하겠습니다. 그로써 
	- "We will now refactor our code, so that it does the same thing as before, ..."
	- we라는 원어가 큰 의미를 가지지 않으므로 번역하지 않아도 될 것 같습니다. 
	- 그리고 원어에서 나온 쉼표를 고려하지 않아도 된다면 

- 함수 뿐만: 함수뿐만
	- '그것만이고 더는 없음'의 뜻을 나타내는 보조사 '뿐'은 앞말과 붙여 씁니다.

- _일반적으로_ 라이브러리의 다른 부분을 사용하여 : 대개는 라이브러리의 다른 부분을 사용하여 
	- "...but as we’ll see, these are **usually** better handled using other parts of the library."
	- "일반적으로"가 적절하지 않은 표현은 아니나 번역체를 다소 부드럽게 하기 위해 대치하는 것도 좋아 보입니다.

- 처리 할 수 : 처리할 수
	- 동작성이 있는 일부 명사나 어근 뒤에 붙어, 그 동작성을 실현하는 동사를 만들거나 또는 상태성이 있는 일부 명사나 어근 뒤에 붙어 '그러한 상태이다'의 뜻을 더하여 형용사를 만드는 접미사 '-하다'는 붙여 씁니다.

- 더이상 : 더 이상
	- 띄어쓰기 오류입니다.

- 확인해봅시다: : 확인해 봅시다:
	- 보조 용언이 두 개 이상 거듭되는 경우 또는 합성동사 뒤에 오는 보조용언은 띄어 쓰는 게 옳습니다.

`## nn.Module 을 이용하여 리팩토링 하기`

- 추척할 수 : 추적할 수
	- 오타로 추정됩니다.

- PyTorch 의 : PyTorch의
	- 이어 나오는 "Python 의", "PyTorch 는"도 조사를 붙입니다.

- 잊어 버리는 : 잊어버리는
	- 표준국어대사전에 등재된 합성어로 붙여 씁니다.
	- [표준국어대사전 - "잊어버리다"](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%EC%9E%8A%EC%96%B4%EB%B2%84%EB%A6%AC%EB%8B%A4)

- 다시 한번 : 한 번
	- 단위를 나타내는 명사는 띄어 씁니다. 
	- "한번"은 과거에 있었던 어느 시점의 일을 이야기할 때를 뜻하는 단어입니다.
	- [표준국어대사전 - "한번"](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%ED%95%9C%EB%B2%88)

`## nn.Linear를 사용하여 리팩토링 하기 ` 전까지만 진행하였습니다.

## 추가 정보
_위와 같이 생각하신 이유 또는 다른 참고할 정보가 있다면 알려주세요._

함께 링크도 첨부한 것을 참고해 주시면 감사하겠습니다. 또한 고민되었던 부분도 첨부합니다. 😄

#### 원어로 동사의 의미를 음차표기한 단어들인데 일관적으로 통일하는 것이 좋아보여서, 번역기에서 각기 다른 결과를 내놓는 예시를 가져왔습니다. (수정 X)  

- 커스터마이즈하기 위해서 : 커스터마이즈 하기 위해서
	- 합성어로 굳어지지 않아 표준국어대사전에 등재되어 있지 않은 명사와 용언은 띄어 쓰는 것이 옳습니다. 
	- [한국어 어문 규범 - 보조용언 띄어쓰기](https://www.korean.go.kr/kornorms/regltn/regltnView.do?regltn_code=0001&regltn_no=232#a267)

- 업데이트 합니다 : 업데이트합니다
	- 동작성이 있는 일부 명사나 어근 뒤에 붙어, 그 동작성을 실현하는 동사를 만들거나 또는 상태성이 있는 일부 명사나 어근 뒤에 붙어 '그러한 상태이다'의 뜻을 더하여 형용사를 만드는 접미사 '-하다'는 붙여 씁니다.
	- [표준국어대사전 - '-하다'2](https://stdict.korean.go.kr/search/searchView.do?pageSize=10&searchKeyword=%ED%95%98%EB%8B%A4)

- 임포트하기 위한 : 임포트 하기 위한
	- 합성어로 굳어지지 않아 표준국어대사전에 등재되어 있지 않은 명사와 용언은 띄어 쓰는 것이 옳습니다.

#### "layer"에 관하여

- layer를 용어집에서는 계층으로 번역하기로 하였다고 하지만 번역본의 선형 레이어, 즉 원어의 linear layers처럼 굳어진 표현이 더 자연스러운 경우도 있다고 생각합니다. 이 부분은 따로 언급하지 않았습니다.
